wurst/_handles/Camera.wurst

```diff
@@ -0,0 +1,166 @@
+package Camera
+import NoWurst
+import public Vectors
+
+public function getCameraBoundMin() returns vec2
+    return vec2(GetCameraBoundMinX(), GetCameraBoundMinY())
+
+public function getCameraBoundMax() returns vec2
+    return vec2(GetCameraBoundMaxX(), GetCameraBoundMaxY())
+
+public function getCameraTargetPositionVec2() returns vec2
```

Comment:
Don't end function names with "vec", it's the default. Position is enough.

Reply:
Changed getCameraTargetPositionVec2 to getCameraTargetPosition and getCameraTargetPositionVec3 to  getCameraTargetPosition3
Same with CameraEye